project/tikv/project.json

```diff
@@ -0,0 +1,108 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+
+    "maintainers": [
+        "BusyJay",
+        "andylokandy",
+        "breeswish",
+        "hicqu",
+        "nrc",
+        "skyzh",
+        "yiwu-arbug"
+    ],
+
+    "committers": [
+        "5kbpers",
+        "AndreMouche",
+        "Connor1996",
+        "Little-Wallace",
+        "MyonKeminta",
+        "NingLin-P",
+        "TennyZhuang",
+        "bobotu",
```

Comment:
Please remove me

Reply:
Thanks for your report. I will add you as an emeritus committer. An emeritus committer won't be requested in code affairs or have effective permission on the repo, but we do respect your works on this project :)